static/app/views/automations/components/automationBuilder.tsx

```diff
@@ -89,6 +91,11 @@ export default function AutomationBuilder() {
             actions.updateWhenCondition(id, comparison)
           }
         />
+        {(mutationErrors as any)?.actionFilters?.all && (
+          <StyledAlert type="error">
+            {(mutationErrors as any).actionFilters.all}
```

Comment:
i couldn't find a better way to get the typing errors to go away 😭  this is because `mutationErrors` is of type `ResponseJSON`

https://github.com/getsentry/sentry/blob/7858f27cbe951d3ac3a3ccec732167df64caada3/static/app/utils/requestError/requestError.tsx#L26-L29